apps/accelerate/speedster/docs/en/docs/installation.md

```diff
@@ -169,14 +171,57 @@ If you want to manually install the requirements, this section collects links to
 
 ## (Optional) Download Docker images with frameworks and optimizers
 
-Instead of installing the frameworks and compilers needed for optimization, which can be a time-consuming task, you can simply download a docker container with all compilers preinstalled.
+Instead of installing the frameworks and compilers needed for optimization, which can be a time-consuming task, you can simply download a Docker container with all compilers preinstalled.
 
-To pull up the docker image, run:
+To pull up the Docker image, run:
 
     docker pull nebulydocker/nebullvm:latest
 
-and then run and access the docker with:
+and then run and access the Docker with:
 
     docker run -ti --gpus=all nebulydocker/nebullvm:latest
 
-After optimizing the model, you may decide to deploy it to production. Note that you need to have the deep learning compiler used to optimize the model and other components inside the production docker. For this reason, we have created several versions of the Docker nebullvm container in the [Docker Hub](https://hub.docker.com/repository/docker/nebulydocker/nebullvm), each containing only one compiler. Pull the image with the compiler that has optimized your model!
\ No newline at end of file
+After optimizing the model, you may decide to deploy it to production. Note that you need to have the deep learning compiler used to optimize the model and other components inside the production Docker. For this reason, we have created several versions of the Docker nebullvm container in the [Docker Hub](https://hub.docker.com/repository/docker/nebulydocker/nebullvm), each containing only one compiler. Pull the image with the compiler that has optimized your model!
+
+## Set up Speedster on custom DL devices
+
+From version `0.10.0`, Speedster supports optimization of PyTorch models on `Google TPUs` and `AWS Inferentia` chips. 
+For these devices, the user must ensure that the required libraries are installed on the machine. 
+The following sections describe how to install the required libraries for each device.
+
+### Google TPUs
+
+In order to use a TPU, you must request a TPU-enabled VM from Google Cloud. You can consult the [official documentation](https://cloud.google.com/tpu/docs/run-calculation-pytorch?hl=en) 
```

Comment:
I imagine here that we are not supporting tf on TPUs right?

Reply:
Yep right, I'll open an issue as soon as we perform the release